addons/website_blog/data/website_blog_demo.xml

```diff
@@ -40,6 +40,7 @@
             <field name="tag_ids" eval="[(6, 0, [ref('blog_tag_1'), ref('blog_tag_2')])]"/>
             <field name="cover_properties">{"background-image": "url('/website_blog/static/src/img/cover_1.jpg')", "resize_class": "o_record_has_cover o_half_screen_height", "opacity": "0"}</field>
             <field name="content"><![CDATA[
+<section class="s_text_block pt16 pb16" data-snippet="s_text_block"><div class="o_container_small"><div>
```

Comment:
I doubt there is a DIV layer without any class in the standard text block snippet